modules/giapi/src/main/scala/giapi/client/package.scala

```diff
@@ -292,6 +292,7 @@ package client {
             IO.sleep(timeout) *>
             IO.pure(CommandResult(Response.COMPLETED))
           } else {
+            IO.sleep(20.seconds) *>
```

Comment:
This will have to be removed, obviously.

Reply:
Not sure, we may keep it shorter, like 5 secs to make the simulator more realistic